### ISSUE TITLE
Geomap map focus

### ISSUE THREAD

**Issue by jolevesq** [wet-boew/wet-boew#2468]
@pjackson28 @jvanulde @sarumim @dfarough @nschonni @mysticjeff This issue is there to continue the discussion started on PR 2401 (https://github.com/wet-boew/wet-boew/pull/2401) about Geomap and accessibility concern.

**Comment by hsrudnicki** [comment]
@dfarough @pjackson28 @jolevesq : can you close this issue or otherwise ?

**Comment by hsrudnicki** [comment]
Please update this issue's status whether this issue can be closed or otherwise?

**Comment by hsrudnicki** [comment]
Please continue the discussion started in 2013, and as well update this issue's status whether this issue is ready to be closed or otherwise?

**Comment by hsrudnicki** [comment]
Due to a lack of further discussion or status update(s), this issue has been labeled "Going Stale" and closed.
